code/__HELPERS/mobs.dm

```diff
@@ -473,3 +473,31 @@ GLOBAL_LIST_EMPTY(species_list)
 		chosen = pick(mob_spawn_meancritters)
 	var/mob/living/simple_animal/C = new chosen(spawn_location)
 	return C
+
+/**
+  * Gives the target carbon mob their items listed in species.items_important_for_life
+  *
+  * Checks all items worn/held/in inventory on mob and gives/equips if they don't already have it.
+  * Tries to equip, put in hands, or drop underneath in that order.
+  */
+/proc/give_important_for_life(var/mob/living/carbon/target)
+	// Path to all items classed as important for a mob to live
+	var/list/important_for_life = target.dna.species.items_important_for_life
+	// Get everything from a mob. Worn/held/in inventory
+	var/list/all_items_on_mob = target.GetAllContents()
+
+	for (var/item_for_life in important_for_life)
+		var/found = FALSE
+		// Try and find if we already have it.

```

Comment:
i think theres procs for this

Reply:
and otherwise, this proc is not that great for what its doing and could be done in /datum/outfit.